### ISSUE TITLE
Add integration tests 

### ISSUE THREAD

**Issue by pociej** [golemfactory/onboarding#41]

**Comment by golmek** [comment]
Testing tool: https://www.cypress.io/ + Xstate

**Comment by golmek** [comment]
Estimated delivery: next week

**Comment by golmek** [comment]
- Blockchain elements (like metamask popup) has to be mockup.

**Comment by golmek** [comment]
- Work in progress: Mocker is not working ... yet :(

**Comment by golmek** [comment]
Parked for Design period

**Comment by golmek** [comment]
Unblocked

**Comment by golmek** [comment]
To be re-visited
